tensorflow/lite/java/demo/app/src/main/java/com/example/android/tflitecamerademo/ImageClassifier.java

```diff
@@ -171,8 +171,15 @@ private void recreateInterpreter() {
   }
 
   public void useGpu() {
+    useGpu(false);
+  }
+
+  public void useGpu(boolean allowQuantizedModels) {
```

Comment:
Do we need this boolean & the method above? We can just always set the param to true?

Reply:
Agreed, let's just always set to true. We'll probably deprecate/remove that flag in a future update and just always allow quantized execution.

Reply:
updated. thanks.